addons/mail/static/src/new/web/thread_service_patch.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module */
+
+import { ThreadService, threadService } from "../core/thread_service";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ThreadService.prototype, "mail/web", {
+    setup(env, services) {
+        this._super(env, services);
+        /** @type {import("@mail/new/web/chat/chat_window_service").ChatWindow} */
+        this.chatWindow = services["mail.chat_window"];
+    },
+    open(thread, replaceNewMessageChatWindow) {
+        if (this.store.discuss.isActive && !this.store.isSmall) {
```

Comment:
It's annoying to have to duplicate this condition here, maybe the patch could be done slightly differently to avoid this.